rollup/plugin-userscript.ts

```diff
@@ -31,8 +31,7 @@ const DEFAULT_OPTIONS = {
 };
 
 function _userscript(options: Readonly<_UserscriptOptionsWithDefaults>): Plugin {
-    const longestMetadataField = [...options.metadataOrder]
-        .sort((a, b) => b.length - a.length)[0];
+    const longestMetadataFieldLength = Math.max(...options.metadataOrder.map((field) => field.length));
```

Comment:
I have to say that I really enjoyed reading (all) your typescript code, you have a really elegant and clean coding style 👍
But this line has been giving me a headache ever since I came across it the first time, so I took the opportunity to improve it while I was already editing this module 😇

Reply:
100% agree. Absolute horrific. I have nothing to say to defend myself. 😄